product-service/src/handlers/addProduct.js

```diff
@@ -0,0 +1,58 @@
+import { Client } from 'pg';
+
+const { PG_HOST, PG_PORT, PG_DATABASE, PG_USERNAME, PG_PASSWORD } = process.env;
+const dbConfig = {
+    host: PG_HOST,
+    port: PG_PORT,
+    database: PG_DATABASE,
+    user: PG_USERNAME,
+    password: PG_PASSWORD,
+    ssl: {
+        rejectUnauthorized: false
+    },
+    connectionTimeoutMillis: 5000
+};
+
+const addProduct = async event => {
+    console.log(`Lambda invocation - addProduct - product: ${event.body} - event: `, event);
```

Comment:
[proposal]
Logging params, catching error code repeats across all lambdas => may be a good idea to create helper function for supporting DRY principle